### ISSUE TITLE
Write metadata to file in folder

### ISSUE THREAD

**Issue by aneurinprice** [manyfold3d/manyfold#2452]
**Is your feature request related to a problem? Please describe.**
It would be really nice to have a files tags, name, links, creator and such stored in a file (yaml or similar) in each models directory. This would be taking more of an "Infrastructure as code" approach and would make a database failiure considerably less painful!

**Describe the solution you'd like**
I'd love a setting that would read/write to/from a `.modelinfo.yaml` to fully contain the models info.

**Describe alternatives you've considered**
Shell scripts/CI jobs that would interact with an API (does not exist yet) to consume this data.

**Additional context**

**Comment by Floppy** [comment]
Yes, I really like this idea, and it's useful to see it come from someone else too. I did some thinking about the package format a while back in #1040. I do like this idea for disaster recovery if the database is lost, as well as distribution of files by creators.

**Comment by aneurinprice** [comment]
I'd be more than happy to help in any way I can for this. Having a standardised metadata setup would help me massively.

**Comment by Floppy** [comment]
We have this now, including reading the data back in during upload and scan!
